hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,16 @@
+RELEASE_TYPE: minor
+
+This release changes the stateful testing backend from
+:func:`~hypothesis.find` to use :func:`@given <hypothesis.given>`
+(:issue:`1300`).  This doesn't change how you create stateful tests,
+but does make them run more like other Hypothesis tests:
+
+- The ``timeout`` setting now applies to stateful tests (:issue:`1213`)
```

Comment:
I don't think this is a good idea - timeout is deprecated. Why would we add in a new deprecated feature?

Reply:
We could go with the status quo and silently ignore the `timeout` setting for stateful tests?  I thought consistency was more elegant but happy to go that way if you prefer.